config/assets.us.yaml

```diff
@@ -248,7 +248,37 @@ files:
         vram: 0x80180000
         assets:
           - [0x2C, sprite_banks, sprite_banks]
-          - [0x90, skip]
+          - [0x90, paldef, palette_def]
+          - [0x150, layers, layers]
+          - [0x5B8, gfx_banks, graphics_banks]
+          #- [0x778, layout, entity_layouts]
```

Comment:
Getting  `unable to extract asset "entity_layouts" in "assets/st/no0": gap between data detected: 0x801839C2 != 0x80183A64` when trying to import this